src/main/java/core/basesyntax/FigureDrawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureDrawable {
+    void drawFigure();
```

Comment:
```suggestion
public interface Drawable {
    void draw();
```
Remember Abstraction.